Code/Editor/EditorPreferencesPageViewportCamera.cpp

```diff
@@ -80,14 +80,18 @@ void CEditorPreferencesPage_ViewportCamera::CameraMovementSettings::Reflect(AZ::
         ->Field("DefaultOrientation", &CameraMovementSettings::m_defaultPitchYaw)
         ->Field("DefaultOrbitDistance", &CameraMovementSettings::m_defaultOrbitDistance)
         ->Field("SpeedScale", &CameraMovementSettings::m_speedScale)
+        ->Field("InstantGoToPosition", &CameraMovementSettings::m_instantGoToPosition)
```

Comment:
very minor suggestion would be to use 
`m_goToPositionInstantly`

Reply:
Updated everything to 'GoToPositionInstantly' 👍